src/AWS/Lambda/Context.hs

```diff
@@ -83,6 +83,11 @@ class HasLambdaContext r where
 instance HasLambdaContext LambdaContext where
   withContext = const
 
+-- TODO: This sticks around for backwards compatibility, and as a conevient-ish
+-- way to runReaderTLambdaContext.  However, long term it would be better if
+-- some of the ergonomics around Reader LambdaContext were improved.  For
+-- example, there _is_ no LambdaContex until the event is received, but our
```

Comment:
s/LambdaContex/LambdaContext/

Reply:
And actually most of this comment is no longer accurate because we don't really use or encourage `Reader LambdaContext`. any more.

Reply:
Good catch, queuing a change now, and I'll push after we find out the result of these builds.